movie_REST_API/movie_REST_API/App/Presenter/InfoMoviePresenter.swift

```diff
@@ -0,0 +1,71 @@
+// InfoMoviePresenter.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+protocol InfoMovieViewProtocol: AnyObject {
+    func succes(data: Data)
+    func failure(error: Error)
+}
+
+protocol InfoMovieViewPresenterProtocol: AnyObject {
+    var movies: Movie { get set }
+    init(
+        view: InfoMovieViewProtocol,
+        networkService: NetworkServiceProtocol,
+        photoLoadService: PhotoLoadServiceProtocol,
+        router: RouterProtocol,
+        movies: Movie
+    )
+    func getImageDataFrom()
+}
+
+/// Презентер экрана с описанием фильмов
+final class InfoMoviePresenter: InfoMovieViewPresenterProtocol {
+    // MARK: - Private Constant
+
+    private enum Constant {
+        static let firstPartURLString = "https://image.tmdb.org/t/p/w500"
+        static let emptyString = ""
+    }
+
+    // MARK: - Public Properties
+
+    let networkService: NetworkServiceProtocol?
+    let photoLoadService: PhotoLoadServiceProtocol?
+    var router: RouterProtocol
+    var movies: Movie
+    weak var view: InfoMovieViewProtocol?
+
+    // MARK: Init
+
+    required init(
+        view: InfoMovieViewProtocol,
+        networkService: NetworkServiceProtocol,
+        photoLoadService: PhotoLoadServiceProtocol,
+        router: RouterProtocol,
+        movies: Movie
+    ) {
+        self.view = view
+        self.networkService = networkService
+        self.photoLoadService = photoLoadService
+        self.router = router
+        self.movies = movies
+        getImageDataFrom()
+    }
+
+    // MARK: - Public Methods
+
+    func getImageDataFrom() {
+        guard let presentImageURLString = movies.presentImageURLString else { return }
+        let url = "\(Constant.firstPartURLString)\(presentImageURLString)"
```

Comment:
```suggestion
        let url = "\(Constant.firstPartURLString)\(presentImageURLString)"
```
Урл компонетс но в апи сервис